homework/v.paslavskiy/Hometask_10.py

```diff
@@ -0,0 +1,30 @@
+def tag(name):
+    def decorator(foo):
+        def wrapper(*args):
```

Comment:
args contains decorated function arguments, so our decorated function (in the end of file) have only one `text` argument. So I thing, you will figure out what you should doing.